src/std/fs.ab

```diff
@@ -70,12 +70,13 @@ pub fun make_executable(path: Text): Bool {
 /// Changes the owner of a file.
 ///
 /// If the file doesn't exist, it returns `false`
-pub fun change_owner(user: Text, path: Text): Bool {
-    if file_exist(path) or dir_exist(path) {
+pub fun file_chown(path: Text, user: Text): Bool {
```

Comment:
Swapped `path` and `user` parameters because all other functions have the filename as the first parameter.